dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaExtension.kt

```diff
@@ -134,11 +132,8 @@ constructor(
      * [Gradle Worker](https://docs.gradle.org/8.5/userguide/worker_api.html).
      *
      * You can control whether Dokka Gradle Plugin launches Dokka Generator in
-     * * a new process, using [ProcessIsolation],
-     * * or the current process with an isolated classpath, using [ClassLoaderIsolation].
-     *
-     * _Aside: Launching [without isolation][WorkerExecutor.noIsolation] is not an option, because
-     * running Dokka Generator **requires** an isolated classpath._
+     * - a new process, using [ProcessIsolation],
```

Comment:
minor: probably it's better to write here WHY someone needs to change this. E.g as in example, regarding memory usage.

Reply:
I've added a lot more detail.